Library/Formula/sngrep.rb

```diff
@@ -0,0 +1,23 @@
+class Sngrep < Formula
+  homepage "https://github.com/irontec/sngrep"
+  url "https://github.com/irontec/sngrep/archive/v0.2.2.tar.gz"
```

Comment:
> Otherwise, 0.2.2 will include them and be closed on tuesday 10th (if no mayor bug appears).

You may not be but just to check: please don't retag releases. It's hard to tell the difference between you doing so and e.g. someone hacking your account and doing so.